src/dev/i18n/extract_default_translations.js

```diff
@@ -126,16 +129,17 @@ export async function extractMessagesFromPathToMap(inputPath, targetMap, config)
       );
 
       for (const { name, content } of files) {
+        const reporterWithContext = reporter.withContext({ name });
+
         try {
-          for (const [id, value] of extractFunction(content)) {
-            validateMessageNamespace(id, name, config.paths);
-            addMessageToMap(targetMap, id, value);
+          for (const [id, value] of extractFunction(content, reporterWithContext)) {
+            validateMessageNamespace(id, name, config.paths, reporterWithContext);
+            addMessageToMap(targetMap, id, value, reporterWithContext);
           }
         } catch (error) {
           if (isFailError(error)) {
```

Comment:
__nit:__ this will help us to eliminate unnecessary `continue` in `catch` clause. Please, make sure you use the same pattern everywhere (I noticed that inconsistency in some other places too, e.g. in `html` and `handlebars`).
```js
if (!isFailError(error)) {
  throw error;
}

reporterWithContext.report(error);
```